src/main/java/org/example/ListExamples.java

```diff
@@ -175,4 +180,13 @@ public static int fibonaccis(int nthTerm) {
         return curr;
     }
 
+
+    public static Map mapConverter(List<String> input) {
```

Comment:
method name should be `countWords`. Also, return type should be `Map<String, Integer>`